reference.md

```diff
@@ -1005,6 +1002,28 @@ macros into smaller scopes as well. Note that macros are a
 compile-time construct; they do not exist at runtime. As such macros
 cannot be exported at the bottom of a module like functions and other values.
 
+Briefly, a macro can be thought to "paste" an espression in its place. A macro
+should return a "quoted" expression (via `` ` ``), to be "pasted". That
```

Comment:
maybe instead of "paste" the "splice" term can be used? I think I saw "spliced in" in some macro docs for other lisps

Reply:
Yeah I didn't super love "paste". Splice is a bit of an unusual term too, technically it's joining or inserting, not replacing.

Maybe just "replace", it's probably most explicit?

> When called, a macro replaces it's own expression with another expression, derived from macro.

Saying "expression" "expression" "derived" isn't so approachable, depending on whether "expression" is a term assumed to be "known" by the reader, but it's probably most clear that `x becomes y`?

>  Briefly, a macro can be thought to splice an expression in its place.

> When called (invoked?), a macro splices an expression in its place.

Also it should be *ex*pression not ~~espresso~~ *es*pression.

Reply:
> Maybe just "replace", it's probably most explicit?

Yeah, probably. My thinking was something like that:

> Briefly, a macro can be thought to splice an expression in its place.

Splice in sounds right, because there's `unquote-splicing` in other Lisps, which ATM is missing in Fennel.

But I guess somenting like "replacing the call of the macro with its returned quoted expression" seems the most clear from the perspective of someone who doesn't know macros in other Lisps.